import.py

```diff
@@ -649,10 +656,77 @@ def write_to_file(cont: str, filename: str) -> None:
         f.write(cont)
 
 
+def download_decompme(url_str: str) -> None:
+
+    try:
+        # unique decomp.me identifier for scratches
```

Comment:
comments on their own lines should be complete sentences. (this one can be removed imo)